### PR TITLE
Emoji picker (#547)

### LINKED ISSUE #547: Implement react-native-emoji-picker
At the moment the emoji icon does nothing, lets replace keyboard with this picker.

https://www.npmjs.com/package/react-native-emoji-picker

### PR DESCRIPTION
**Pull request by alwx**
Fixes #547

### CODE REVIEW

**Comment by jarradh** [comment]
`lein deps && npm install && ./re-natal deps && lein prod-build && react-native run-android --variant=release` didn't seem to install the rn emoji package, i had to follow up with `npm i react-native-emoji-picker --save`

Also this doesn't replace the keyboard, in other words the emoji picker should be under the text input field. Currently it's possible to display keyboard and this on top.

**Comment by alwx** [comment]
It has been updated.
screen shot 2016-12-20 at 18 07 13

File: src/status_im/ios/core.cljs [hunk 1]
```diff
@@ -64,7 +64,8 @@
                        (fn [e]
                          (let [h (.. e -endCoordinates -height)]
                            (when-not (= h @keyboard-height)
-                             (dispatch [:set :keyboard-height h])))))
+                             (dispatch [:set :keyboard-height h])
+                             (dispatch [:set :keyboard-max-height h])))))
```
Comment (alwx) [inline src/status_im/ios/core.cljs]: No. 
`:keyboard-height` is the actual height and it can be 0 if the keyboard is hidden.

**Comment by rasom** [comment]
Is it fine that images are a bit cut off on android?
image

**Comment by alwx** [comment]
@rasom I guess it is better to ask this question to `react-native-emoji-picker` developer, but I don't see the same problem on my Android device. I guess it is somehow related to the manufacturer and the emojis you have on your device.

**Comment by jarradh** [comment]
I'm not concerned about us having to worry about the emoji size in the picker for alpha

**Pull request merged**